contracts/token/SRC20.sol

```diff
@@ -325,40 +326,40 @@ contract SRC20 is ISRC20, ISRC20Owned, ISRC20Managed, SRC20Detailed, Featured,
      *
      * @return True if token is frozen.
      */
-    function isTokenFrozen() external view returns (bool) {
-        return _isTokenFrozen();
+    function isTokenPaused() external view returns (bool) {
```

Comment:
I don't see anywhere that transfers are forbidden on paused token. If this is to be the case, please add tests that will check for this condition.

Reply:
@tomislavr The check is already their.
https://github.com/swarmfund/swarm-compliant-contract/blob/6497f3a1fa2416bc7637297a7a369e7c5fad9fae/contracts/token/SRC20.sol#L467

Reply:
oh, it is in a separate branch, pardon me.